### PR TITLE
Homework JS2 week3

### PR DESCRIPTION
**Pull request by veronicaburduzhan**
Thanks in advance for your comments and review:)

### CODE REVIEW

File: javascript/javascript2/week3/homework/warmup.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+const earthLogger = () => {
+  console.log("Earth");
+};
+
+const saturnLogger = () => {
+  console.log("Saturn");
+};
+
+function planetLogFunction(callback) {
+  callback(planetLogFunction);
```
Comment (fathermatti) [inline javascript/javascript2/week3/homework/warmup.js:43]: No need to pass planetLogFunction to the callback function :)

File: javascript/javascript2/week3/homework/warmup.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      lng: geolocationPosition.coords.longitude,
+    },
+    zoom: 8,
+  }
+};
+
+// 7. Function with 2 parameters: delay and callback
+function runAfterDelay(delay, callback) {
+  setTimeout(() => {
+    callback(runAfterDelay);
```
Comment (fathermatti) [inline javascript/javascript2/week3/homework/warmup.js:87]: Same

File: javascript/javascript2/week3/homework/warmup.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+}
+
+//3. Object of function
+const funcObject = {
+  foo: function () {
+    console.log("Hi! I'm in object");
+  },
+};
+
+funcObject.foo();
```
Comment (fathermatti) [inline javascript/javascript2/week3/homework/warmup.js:164]: Super smooth solving of all questions - a shame with the Google Maps stuff, but it was optional anyway :)

File: javascript/javascript2/week3/homework/fastest-presser.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+const timerInput = document.getElementById("timer");
```
Comment (fathermatti) [inline javascript/javascript2/week3/homework/fastest-presser.js:1]: Solution looks good and works mostly as intended, well done! It handles wins, draws, missing input and no presses.

The replay logic is a little shaky - it does not always let me replay, and it says "Play Game" and not "Play Again" when someone has won. But, those are minor things, overall well done!

Comment (veronicaburduzhan) [reply javascript/javascript2/week3/homework/fastest-presser.js:1]: Thanks for your review!:)

**Pull request merged**